spec/datadog/appsec/waf/handle_spec.rb

```diff
@@ -21,6 +21,18 @@
             }
           ],
           'action' => 'record'
+        },
+        {
+          'id' => 2,
```

Comment:
I have to add another rule here, otherwise the test on line 193 will fail since we are disabling the only existing rule, and the WAF Handle cannot be created without rules after [this change](https://github.com/DataDog/libddwaf/pull/314/files#diff-d067197e9befed2abc2cac3d47e95a647139db7939bd1a1334a1b51a4c257864R198-R201)